sdmetrics/single_table/new_row_synthesis.py

```diff
@@ -86,6 +86,25 @@ def compute_breakdown(cls, real_data, synthetic_data, metadata=None,
         except IncomputableMetricError:
             categorical_fields = []
 
+        def update_column_name(char, column_name):
+            return column_name.replace(char, '_') if char in column_name else column_name
+
+        def update_lists(column, new_column):
```

Comment:
since this method is nested and only used here, I think we can just call the for loop directly and get rid of it